tempo/serve/stub.py

```diff
@@ -0,0 +1,48 @@
+import json
+import os
+from tempfile import NamedTemporaryFile, TemporaryDirectory
+from urllib.parse import urlparse
+
+import rclone
+
+from tempo.serve.base import BaseModel, RemoteModel
+from tempo.serve.runtime import ModelSpec
+
+from ..conf import settings
+from ..utils import logger
+
+
+def _load_rclone_cfg() -> str:
+    with open(settings.rclone_cfg, "r") as f:
+        return f.read()
+
+
+def deserialize(d: dict) -> RemoteModel:
```

Comment:
deserialize sounds like it will load from pickle, perhaps we could just call it model_from_spec?

Reply:
Well I guess serialize / deserialize is actually correect, so we can probably stick with this

Reply:
ok